glui.cpp

```diff
@@ -210,7 +264,17 @@ void GLUI_Main::setup_default_glut_callbacks( void )
   glutPassiveMotionFunc( glui_passive_motion_func );
   glutEntryFunc( glui_entry_func );
   glutVisibilityFunc( glui_visibility_func );
-  /*  glutIdleFunc( glui_idle_func );    // FIXME!  100% CPU usage!      */
+  /*  glutIdleFunc( glui_idle_func );    // FIXME!  100% CPU usage!      */  
+  //This is needed to host the X11 copy-paste selection-owner.
+  //It's really no big deal. The scheduler will divide time up
+  //and unless "vsync" is off a continuously rendering client
+  //will wait for the "vertical-blank."
+  //GLUT isn't a good fit for passive applications, and it is
+  //historically for full-motion 3-D displays.
+  //Clients can figure out a wait solution in their Idle function.
+  //freeglut provides a default idle, which it does not expose,
+  //but it's open-souce and so copy-friendly.
+  glutIdleFunc( glui_idle_func );
```

Comment:
P.S. I would be surprised if classic GLUT does anything whatsoever if the idle function is not provided to it.

I don't think _I will purposely not define an idle function because I believe the library will provide me with a default that is superior, and only because I don't require to do anything in my idle function_ is a defensible position. GLUT doesn't provide any defaults for any callbacks.